bin/cli.js

```diff
@@ -35,11 +35,29 @@ function checkLocalModule(env) {
   }
 }
 
-function initKnex(env) {
+function mkConfigObj(env, commander) {
+  env.configuration = {
+    ext: 'js',
+    [commander.env || process.env.NODE_ENV || 'development']: {
```

Comment:
this is very difficult to read. Could you please split it somewhat? Inline condition inside key name resolution is heavy :)